### ISSUE TITLE
New Plugin: IdentityPlugin: Allow IdentityWidget to work without model but with XML in/out

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#241]
```
To allow reuse of the identitywidget, add to the interface and code the ability 
to only work with XML in/out and/or QAbstractItemModel with an automapping 
management.
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 12 Dec 2012 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
We need to create a specific plugin to avoid complexe plugins dependencies 
(zip, patient, views, core).
We can then include this widget in:
- identity basewidget formitem (specially use XML in/out members)
- usermanager user identity
- patient identity
- (future) contact identity

```

Original comment by `eric.mae...@gmail.com` on 12 Dec 2012 at 11:54
- Changed title: **New Plugin: IdentityPlugin: Allow IdentityWidget to work without model but with XML in/out**

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 00162f7e65f9.
```

Original comment by `eric.mae...@gmail.com` on 12 Dec 2012 at 5:46

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 6f182444b385.
```

Original comment by `eric.mae...@gmail.com` on 12 Dec 2012 at 5:46

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 2de23066ca1b.
```

Original comment by `eric.mae...@gmail.com` on 12 Dec 2012 at 5:46

**Comment by GoogleCodeExporter** [comment]
```
This issue was updated by revision 2898ecb9d96b.
```

Original comment by `eric.mae...@gmail.com` on 12 Dec 2012 at 5:46

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c8e05eaa4814.
```

Original comment by `eric.mae...@gmail.com` on 12 Dec 2012 at 5:46
- Changed state: **Fixed**
